vite.config.ts

```
@@ -0,0 +1,154 @@
+      },
+      UnoCSS(),
+      AutoImport({
+        imports: ['vue', 'uni-app'],
+        dts: 'src/types/auto-import.d.ts',
+        dirs: ['src/hooks'], // 自动导入 hooks
+        eslintrc: { enabled: true },
+        vueTemplate: true, // default false
+      }),
+
+      ViteRestart({
+        // 通过这个插件，在修改vite.config.js文件则不需要重新运行也生效配置
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)